provider-ci/internal/pkg/templates.go

```diff
@@ -39,21 +41,21 @@ func getTemplateDirs(templateName string) ([]TemplateDir, error) {
 	switch templateName {
 	case "bridged-provider":
 		// Any Pulumi-owned bridged provider
-		return []TemplateDir{base, internal, bridged, internalBridged}, nil
+		return []TemplateDir{base, internal, bridged, internalBridged, mise}, nil
```

Comment:
In terms of layering, I think "mise" should go after "base", but before the more specific layers - in case they need to override the mise file.

```suggestion
		return []TemplateDir{base, mise, internal, bridged, internalBridged}, nil
```